lib/index.js

```diff
@@ -508,9 +527,14 @@ async function run(options = {}) {
         const packages = await previousPromise
         // copy object to prevent share .ncurc options between different packageFile, to prevent unpredictable behavior
         const rcResult = getNcurc({ packageFile })
+        let rcConfig = rcResult && rcResult.config ? rcResult.config : {}
+        if (options.recursive) {
+          // inherit parent config options
+          rcConfig = mergeOptions(options, rcConfig)
```

Comment:
This inherits not just the parent's config options, but the parent's command-line options. We currently maintain this distinction:

- command line options apply to every package
- config options apply locally to the package they are in

Merging command-line options instead of just merging config options might have unintended consequences.

Reply:
Yes, you right. Will change implementation to merge onlu subset of keys from `rcConfig`

Reply:
That's not quite what I had in mind. I will try to explain in a review comment!